android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -187,10 +189,16 @@ public void onClick(View v) {
             Timber.d(date+"\n"+endTime+"\n"+startTime);
 
         }
-        summary.setText(session.getSummary());
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
+            summary.setText(Html.fromHtml(session.getSummary(), FROM_HTML_MODE_LEGACY));
+        } else {
+            summary.setText(Html.fromHtml(session.getSummary()));
+        }
+
+        summary.setMovementMethod(LinkMovementMethod.getInstance());
 
         if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
-            result = Html.fromHtml(session.getDescription(), Html.FROM_HTML_MODE_LEGACY);
+            result = Html.fromHtml(session.getDescription(), FROM_HTML_MODE_LEGACY);
```

Comment:
@harshitagupta30 Yes, I have resolved the issue and I haven't removed result variable because it is getting used further(In share action). I noticed that in share action share string should be description but it is always "To be decided soon". So I will solve this issue in another issue. I will create another issue and solve it properly.
Do let me know if I need to make any further changes in this PR.